UPGRADE-6.1.md

```diff
@@ -17,6 +17,7 @@ HttpKernel
 ----------
 
  * Deprecate StreamedResponseListener, it's not needed anymore
+ * Deprecate `ConfigurableExtension` class, use `MicroExtension` instead
```

Comment:
I don't think it should be deprecated. It would make it hard to support several major versions of Symfony in third party bundles.

Reply:
Hmm, you're right, we should do it when 6.4 is the min LTS version available.